scripts/testnet/sovereignBridge/config/esdt-safe.snippets.sh

```diff
@@ -0,0 +1,168 @@
+ESDT_SAFE_ADDRESS=$(mxpy data load --partition=${CHAIN_ID} --key=address-esdt-safe-contract)
+ESDT_SAFE_ADDRESS_SOVEREIGN=$(mxpy data load --partition=sovereign --key=address-esdt-safe-contract)
+
+deployEsdtSafeContract() {
```

Comment:
All the functions for sovereign are suffixes with `Sovereign` because it's easier when you type in console.
If someone wants to do this step by step they will use a guide not guess the order for the functions. Maybe I will add a comment above the functions